plugins/scaffolder-backend/src/service/router.ts

```diff
@@ -498,6 +450,7 @@ export async function createRouter(
         secrets: {
           ...req.body.secrets,
           backstageToken: token,
+          initiatorCredentials: JSON.stringify(credentials),
```

Comment:
@benjdlambert do you think we should consider communicating this at the top of the object instead of within secrets?

Reply:
Hmm depends. If they're sensitive data I'd prefer to keep them in the secrets object and be handled as such like the `backstageToken`. Being deleted when the task is finished for example, rather than having separate handling for that.

Reply:
Settled on having it in secrets behind a prefixed key and keeping it out of the type. Same reasoning, it's nice to have all secrets handling in one place